### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by Decagon**
Small linguistic changes, and the `effect` table has a new `Required?` column (instead of appending the word "optional" to every item.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -583,7 +586,7 @@ Occurs when someone leaves the world.
 Occurs when you liked the world.
 
 ### <a id="rm-lobbyPreviewEnabled">"lobbyPreviewEnabled"</a>
-Occurs when lobby preview is enabled or disabled.
+Occurs when the lobby preview is changed.
```
Comment (Tunous) [inline README.md]: This is not clear. Someone could wonder what does the lobby preview change mean.

File: README.md [hunk 2]
```diff
@@ -1112,7 +1114,7 @@ Sent to request adding of the world to a crew.
 | `0`   | `String` | Crew Id | The crew id. |
 
 ### <a id="sm-reset">"reset"</a>
-Sent to reset progress.
+Sent to add a reset progress block.
```
Comment (Tunous) [inline README.md]: Wrong. The is sent to reset player's stats/progress when pressing Y while on reset block, not to place the block.

File: README.md [hunk 3]
```diff
@@ -1025,10 +1027,10 @@ Sent to touch a hologram.
 
 
 ### <a id="sm-init">"init"</a>
-Sent to request initialization message.
+Sent to request the initialization message.
 
 ### <a id="sm-init2">"init2"</a>
-Sent to request initialization messages such as `add` and `k` events, etc.
+Sent to request the initialization messages such as `add` and `k` events, etc.
```
Comment (Tunous) [inline README.md]: Perhaps we could also link to these messages:
Sent to request the initialization messages such as ["add"](#rm-add) and ["k"](#rm-k), etc.

*(events word is not needed here)*

File: README.md [hunk 4]
```diff
@@ -372,27 +376,26 @@ Occurs when an attempt to request adding of the world to a crew didn't succeed.
...
-| Id  | Type    | Name                  | Description                                                            |
-| --- | ----    | -----                 | -----------                                                            |
-| `0`   | `Integer` | Player Id             | The player's id.                                                       |
-| `1`   | `Integer` | Effect                | The effect's id. *See [Effects](#model-effects).*                      |
-| `2`   | `Boolean` | Enabled               | Value indicating whether the effect is enabled.                        |
-| `3`   | `Double`  | Argument *(optional)* | The optional argument of the effect (time left, number of jumps, etc.) |
-| `4`   | `Integer` | Duration *(optional)* | The duration of the effect.                                            |
+| Id  | Type      | Name      | Required?   | Description                                                            |
+| --- | ----      | -----     | ----------- | --------------                                                         |
+| `0` | `Integer` | Player Id | Required.   | The player's id.                                                       |
```
Comment (Tunous) [inline README.md]: Wouldn't it look better without full stop after *Required*?

File: README.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # Everybody Edits Messages Protocol
-The description of the Everybody Edits game protocol.
+This repo contains documentation on the PlayerIO based (http://everybodyedits.com)[Everybody Edits] API.
```
Comment (Tunous) [inline README.md]: This repo**sitory** contains documentation on the PlayerIO based (http://everybodyedits.com)[Everybody Edits] API.

File: README.md (hunk 4 above)
Comment (Decagon) [inline README.md]: Yeah it would look better as it would match up with the other columns

File: README.md (hunk 5 above)
Comment (Decagon) [inline README.md]: Totally forgot to fix that shorthand. Will fix

File: README.md (hunk 3 above)
Comment (Decagon) [inline README.md]: Linking them would be a good idea, as it adds context. I'll see if there are a few more that I can link up too.

**Comment by Decagon** [comment]
Should be fixed up now. Let me know if there's anything that I might have missed!

**Comment by Tunous** [comment]
Looks good now. Thanks for the contributions :+1:

**Comment by Decagon** [comment]
No problem.

**Pull request merged**